perk-store-services/src/main/java/org/exoplatform/perkstore/service/PerkStoreService.java

```diff
@@ -307,6 +307,8 @@ public Product saveProduct(Product product, String username) throws Exception {
       }
     }
     List<ProductOrder> orders = null;
+    Boolean isPerkStoreManager = isPerkStoreManager(username);
+    filter.setMyOrders(!isPerkStoreManager);
```

Comment:
If this parameter ( `isMyOrders` ) will not be provided by UI, it should be deleted from filter and passed in parameters in DAO layer call outside passed filter DTO.